libs/cart/src/selectors/cart.selector.ts

```diff
@@ -3,32 +3,39 @@ import { DaffCartOrderResult } from '../models/cart-order-result';
 import { DaffCartFeatureMemoizedSelectors, getDaffCartFeatureSelector } from './cart-feature.selector';
 import { DaffCartOrderMemoizedSelectors, getCartOrderSelectors } from './cart-order/cart-order.selector';
 import { DaffCartStateMemoizedSelectors, getCartSelectors } from './cart/cart.selector';
+import { getDaffCartItemEntitiesSelectors, DaffCartItemEntitiesMemoizedSelectors } from './cart-item-entities/cart-item-entities.selectors';
+import { DaffCartItem } from '../models/cart-item';
 
 export interface DaffCartMemoizedSelectors<
   T extends DaffCart = DaffCart,
-  V extends DaffCartOrderResult = DaffCartOrderResult
+	V extends DaffCartOrderResult = DaffCartOrderResult,
+	U extends DaffCartItem = DaffCartItem
 > extends DaffCartFeatureMemoizedSelectors<T, V>,
 	DaffCartOrderMemoizedSelectors<V>,
-	DaffCartStateMemoizedSelectors<T> {}
+	DaffCartStateMemoizedSelectors<T>,
+	DaffCartItemEntitiesMemoizedSelectors<U> {}
 
 const createCartSelectors = <
   T extends DaffCart = DaffCart,
-  V extends DaffCartOrderResult = DaffCartOrderResult
+	V extends DaffCartOrderResult = DaffCartOrderResult,
+	U extends DaffCartItem = DaffCartItem
 >(): DaffCartMemoizedSelectors<T> => {
 
 	return {
 		...getDaffCartFeatureSelector<T, V>(),
 		...getCartOrderSelectors<T, V>(),
-		...getCartSelectors<T>()
+		...getCartSelectors<T>(),
```

Comment:
Pass generics into the selector getters.

Reply:
done